test_collections/matter/sdk_tests/support/python_testing/sdk_python_tests.py

```diff
@@ -170,26 +177,32 @@ def sdk_mandatory_python_test_collection(
 
 def custom_python_test_collection(
     python_test_folder: SDKTestFolder = CUSTOM_PYTHON_TEST_FOLDER,
+    tests_file_path: Path = CUSTOM_PYTHON_TESTS_PARSED_FILE,
 ) -> Optional[PythonCollectionDeclaration]:
+    asyncio.run(
+        generate_python_test_json_file(
+            test_folder=CUSTOM_PYTHON_SCRIPTS_FOLDER,
```

Comment:
Are `SDK_PYTHON_TEST_FOLDER` and `CUSTOM_PYTHON_TEST_FOLDER` (declared in this file) still being used? If not, please delete them.
Maybe I misunderstood, but I think these are the old folders and now both folder are defined in `list_python_tests_classes` and we don't need these anymore.